less/home-page.less

```diff
@@ -58,12 +87,24 @@
       }
 
       .btn {
+        .custom-btn(20%, 35px, @bg-color, @font-color, 1.4rem);
+        @media @mobile {
+          // display: flex;
+          // justify-content: center;
+          width: 100%;
+        }  
+        
```

Comment:
Really good job using both a parametric mixin and variables at the same time here